### PR TITLE
MM-58495: message priority tracking plan

### PR DESCRIPTION
**Pull request by ifoukarakis**
#### Summary

Track message priority feature usage.

- [x] Replace seed files with new ones allowing attribution for properties (only in events).
- [x] Simplify seed files by removing aliases.
- [x] Add `events` table in sources and staging.
- [x]  Use `events` table for calculating feature usage, as it's the one containing properties.
- [x] Revise logic for attributing events to features.
- [x] Mark events indicating message priority feature usage.
- [x] Revise calculation of feature usage.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-58495

### CODE REVIEW

File: transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__event.sql [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+        -- Common event columns
+        id               as event_id
+        , event          as event_table
+        , event_text     as event_name
+        , category
+        , type          as event_type
+        , user_id        as server_id
+        , user_actual_id as user_id
+        , received_at    as received_at
```
Comment (catalintomai) [inline transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__event.sql:20]: nit: only alias what is actually renamed.

File: transform/mattermost-analytics/models/marts/product/features/_product__features__models.yml [hunk 2]
```diff
@@ -66,19 +75,14 @@ models:
           The number of unique users captured the current day and past 29 days performing an action that cannot (yet) 
           be mapped to a feature.
 
-      - name: count_known_features_events_daily
```
Comment (catalintomai) [inline transform/mattermost-analytics/models/marts/product/features/_product__features__models.yml:69]: I believe these need to be added to the new file as well (referenced by tests)

File: transform/mattermost-analytics/models/intermediate/product/features/int_feature_daily_spine.sql [hunk 3]
```diff
@@ -6,21 +6,49 @@
     })
 }}
 
-{# List of known features #}
-{%- set known_features = dbt_utils.get_column_values(ref('feature_aliases'), 'alias') -%}
 
-{%- set unknown_feature ='unknown' -%}
+{%
+    set count_feature_columns = dbt_utils.get_filtered_columns_in_relation(
```
Comment (catalintomai) [inline transform/mattermost-analytics/models/intermediate/product/features/int_feature_daily_spine.sql:11]: is this being used?

**Review by catalintomai — APPROVED** [review]

File: transform/mattermost-analytics/models/marts/product/features/_product__features__models.yml (hunk 2 above)
Comment (ifoukarakis) [reply transform/mattermost-analytics/models/marts/product/features/_product__features__models.yml:69]: The have been readded.

**Pull request merged**